contrib/terraform/openstack/variables.tf

```diff
@@ -102,6 +102,7 @@ variable "packages" {
   default = [
     "kernel-default",
     "-kernel-default-base",
+    "-docker",
```

Comment:
Oh, I completely missed this section when reviewing your previous OpenStack PR.

I think it's completely uncnecessary. We install and remove packages in guestfs scripts. Just remove docker there - in `contrib/guestfs/provision/provision.sh`

Because with that change, we are installing that package in guestfs just to remove it with cloud-init later...

And the proper kernel packages are also installed/removed in the same guestfs script. I think you could easily just remove the whole `packages` variable from cloud-init.

Reply:
I moved this part to https://github.com/rancher-sandbox/lockc/pull/45/commits/2087f4d7f3ebfbb84ea9cf9916dcf6c44d32477e

I would prefer to keep packages feature in terraform scripts, it might be useful in future.

Thank you,

Reply:
Ok, that works for me